pages/AI-resource.md

```diff
@@ -1 +1,7 @@
-![loadingGif](https://ik.imagekit.io/1cw2zpbjy/OSWH/OSWH.gif?ik-sdk-version=javascript-1.4.3&updatedAt=1670220591394)
\ No newline at end of file
+# AI Resources
+
+## Prerequsites
+
+|         Description          |                                                                                                             Preview                                                                                                             |                                  Link to get started with it                                   |
+| :--------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------: |
+| 1. Prerequisites and Prework | <center> [![Prerequisites and Prework](https://developers.google.com/static/machine-learning/images/intro-with-1_960.png "Introduction to ML And AI")](https://www.youtube.com/watch?v=BmDricQGK6w "Introduction to ML and AI") <center> | [Here you go](https://developers.google.com/machine-learning/crash-course/prereqs-and-prework) |
```

Comment:
Please edit the thumbnail and set the image resolution to 320wx180h, For editing the image, you can use canva or any other tools